src/main/java/com/passbook/sparkeighteen/peristence/POJO/ProfileResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.passbook.sparkeighteen.peristence.POJO;
+
+import com.passbook.sparkeighteen.peristence.entity.ProfileEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProfileResponse {
+
+    private ProfileEntity profile;
```

Comment:
We should never pass Entity in response (ProfileEntity)
Add al the fields manually.
Let it be repetitive now.

Reply:
There was a human error we rectified it.